packages/orbit-components/src/InputField/__tests__/index.test.js

```diff
@@ -45,7 +45,7 @@ describe("InputField", () => {
         }}
       />,
     );
-    const input = screen.getByRole("textbox", { name: /Label/ });
+    const input: any = screen.getByRole("textbox", { name: /Label/ });
```

Comment:
Related to the `userEvent.tab` suggestion.
```suggestion
    const input = screen.getByRole("textbox", { name: /Label/ });
```